packages/joint-core/types/joint.d.ts

```diff
@@ -239,7 +239,7 @@ export namespace dia {
 
         getCommonAncestor(...cells: Cell[]): Element | undefined;
 
-        toJSON(): any;
+        toJSON(opt?: { ignoreDefaults: boolean | string[], ignoreEmptyAttributes: boolean }): any;
```

Comment:
Can we create `dia.Cell.ExportOptions` type and use it for the graph method too?